.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build Docker image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_run:
+    workflows: [Lint]
+    types:
+      - completed
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      PYTHON_VERSION: '3.10'
+
+    steps:
+      - name: Checks out repository
+        uses: actions/checkout@v3
+
+      - name: Build Docker image
+        run: docker build . -t magnetic-manticores
+
+      - name: Publish Docker image
+        run: 'echo "TODO: we need to decide where to publish, or maybe build it on VPS directly?"'
```

Comment:
Apologies if this is a stupid question (I haven't worked with Docker or deployment like this before and am a Windows user 🙃), but could this just be via GitHub's release tags?

Reply:
it definitely can be. I mean - if we will publish some tag, build will be done. I did it simpler for now to just have every new push built as new Docker image. We don't know what we will develop yet, but finally - we should do a release on GH and it will trigger new deployment also.